File: Library_python /Video.py
from Media import Media

class Video(Media):
   def __init__(self, call_no, title, subjects, description, 
                distibutor, notes, series, label):
      super().__init__(call_no, title, subjects)
      self.description = description
      self.distributor = distibutor
      self.notes = notes
      self.series = series
      self.label = label

   def display(self):
      super().display()
      print ( "Description: ", self.description, "Distributor: ", self.distributor, "Notes: ", self.notes,"Series: ", self.series,"Label: ", self.label)
   
   def compareByOther(self, search):
      if search in self.description or \
         search in self.distributor or \
         search in self.notes or \
         search in self.series or \
         search in self.label:
         return True
      else:
         return False

File: Library/video.cpp
#include<string>
#include<iostream>
#include"video.h"

Video::Video(string call_no, string title, string subjects,string description,string distributor,string notes,string series,string label):Media(call_no,title,subjects,notes), m_description(description),m_distributor(distributor),m_series(series),m_label(label)
{
}
void Video::display() const
{
    Media::display();
    cout<<"Description: "<<m_description;
    cout<<"Distributor: "<<m_distributor;
    cout<<"Series: "<<m_series;
    cout<<"Label: "<<m_label << endl;
}
bool Video::comp_other(const string& str) const
{
    if(Media::m_notes.find(str) != string::npos ||m_description.find(str) != string::npos || m_distributor.find(str) != string::npos || m_series.find(str) != string::npos || m_label.find(str) != string::npos)
    {
        return true;
    }
    else
    {
        return false;
    }
}
